src/openrct2/scripting/ScRide.hpp

```diff
@@ -624,7 +624,6 @@ namespace OpenRCT2::Scripting
             }
             return ToDuk(ctx, nullptr);
         }
-
```

Comment:
Restore this line.

Reply:
@Gymnasiast Sure. The files are a bit inconsistent though, some getters and setters have no empty line separating them, some do. I removed this one because at first I though this one was the only inconsistent one, but I'm seeing more now.

But if you want it back I can add it back. :)

Reply:
Other properties have no empty line between the getters and setters, probably done on purpose. This change makes the value property more consistent with the rest.